docs/products/clickhouse/howto/use-query-editor.rst

```diff
@@ -19,3 +19,8 @@ Count rows::
 Create a new role::
 
     CREATE ROLE accountant
+
+Alternatives
+-------------
+
+Query editor is convenient if you want to run queries directly from the console on behalf of the default user. However, if you want to run requests using different user, or if you expect a large size of the response, you can use :doc:`play <use-play>`, a built-in user interface accessible through HTTPS.
```

Comment:
The query editor is...

Note: Added "the"

However, if you want to run requests using a different user...

Note: Added "a".